### ISSUE TITLE
BuildError with "tldr" package

### ISSUE THREAD

**Issue by tobozo** [pastelsky/bundlephobia#98]
## Type : Bug
The parser doesn't seem to like the '#' in `#!/usr/bin/env/node`

## Package name
tldr ([github project](https://github.com/tldr-pages/tldr-node-client))

### Entire error (stringified) I see in my browser console
```
{
  "error": {
    "code": "BuildError",
    "message": "Failed to build this package.",
    "details": {
      "name": "BuildError",
      "originalError": [
        "/tmp/tmp-build/packages/build-tldr@3.2.5/node_modules/tldr/bin/tldr\nModule parse failed: Unexpected character '#' (1:0)\nYou may need an appropriate loader to handle this file type.\n| #!/usr/bin/env node\n| \n| const program = require('commander');\n @ /tmp/tmp-build/packages/build-tldr@3.2.5/index.js 1:11-26"
      ]
    }
  }
}
```

**Comment by pastelsky** [comment]
Support for `#!` in `.js` files is still in one of the `stage-X` proposals. I don't think its standard syntax, hence the error.

**Comment by daKmoR** [comment]
is there any way to get the size of a bin/cli package then?

examples:
https://bundlephobia.com/result?p=create-react-app
https://bundlephobia.com/result?p=create-open-wc

```
/tmp/tmp-build/packages/build-create-react-app@2.1.5-nUS/node_modules/create-react-app/index.js 1:0 Module parse failed: Unexpected character '#' (1:0) You may need an appropriate loader to handle this file type. > #!/usr/bin/env node | | /** @ /tmp/tmp-build/packages/build-create-react-app@2.1.5-nUS/index.js 1:10-37
```

For open-wc it's actually a really "simple" file
https://github.com/daKmoR/create-open-wc/blob/master/src/create-open-wc.js

the `#!/usr/bin/env node` only makes sure you can run it via bash like so `npm init open-wc`.
According to the npm documentation it's absolutely needed.
> Please make sure that your file(s) referenced in bin starts with #!/usr/bin/env node, otherwise the scripts are started without the node executable!

https://docs.npmjs.com/files/package.json#bin

### Possible Workaround:
For the build process it would be perfectly fine to just remove this line as it will not affect the bundle size.

If you could point me to the correct file/folder I could make a PR.
